contracts/bridge/Bridge.sol

```diff
@@ -123,27 +141,33 @@ contract Bridge is ITokenReceiver, INFTReceiver, Ownable {
         _to.transfer(_amount);
     }
 
-    // handleNFTTransfer sends the NFT by the request.
-    function handleNFTTransfer(
+    // handleERC721Transfer sends the NFT by the request.
+    function handleERC721Transfer(
         uint256 _uid,
         address _to,
         address _contractAddress,
         uint64 _requestNonce,
-        uint64 _requestBlockNumber
+        uint64 _requestBlockNumber,
+        string _tokenURI
     )
         external
         onlyOwner
     {
         require(handleNonce == _requestNonce, "mismatched handle / request nonce");
 
-        emit HandleValueTransfer(_to, TokenKind.NFT, _contractAddress, _uid, handleNonce);
+        emit HandleValueTransfer(_to, TokenKind.ERC721, _contractAddress, _uid, handleNonce);
         lastHandledRequestBlockNumber = _requestBlockNumber;
         handleNonce++;
-        IERC721(_contractAddress).safeTransferFrom(address(this), _to, _uid);
+
+        if (modeMintBurn) {
+            ERC721MetadataMintable(_contractAddress).mintWithTokenURI(_to, _uid, _tokenURI);
```

Comment:
`modeMintBurn` and `metaMintable` seem to mean different things, but do you have any reasons to treat them with the same variable condition `modeMintBurn`?

Reply:
To mint token, the ERC721 should be mintable. Also This PR supports URI (metadata) migration. So when token is mint, URI is needed.